DiscordBot/Commands/EventCommands.cs

```diff
@@ -384,18 +384,17 @@ private async Task CreateSignupSheet(CommandContext context, int eventKey)
                 return;
             }
 
-            try
-            {
-                var signupChannel = context.Guild.Channels
-                    .First(channel => channel.Value.Name == SignupChannelName)
-                    .Value;
+            var signupChannel = context.Guild.Channels
+                .FirstOrDefault(channel => channel.Value.Name == SignupChannelName)
+                .Value;
```

Comment:
I don't think you can call `Value` here if null has been returned? Have you tested this for a channel which doesn't exist? 🙂

Reply:
I had that same thought and wanted to use ?.Value but the IDE didn't allow it and it appears to be working when the channel doesn't exist.
![image](https://user-images.githubusercontent.com/36454654/99816456-9e165900-2b43-11eb-846d-e5c9314fdca5.png)

Reply:
It seems the default value is a key value pair of [0, null].

Reply:
Ah I see 🙂